test/jdk/java/lang/Thread/virtual/SynchronizedNative.java

```diff
@@ -28,31 +28,31 @@
  * @requires vm.continuations
  * @modules java.base/java.lang:+open jdk.management
  * @library /test/lib
- * @run junit/othervm/native --enable-native-access=ALL-UNNAMED SynchronizedNative
+ * @run junit/othervm/native/timeout=300 --enable-native-access=ALL-UNNAMED SynchronizedNative
  */
 
 /*
  * @test id=Xint
  * @requires vm.continuations
  * @modules java.base/java.lang:+open jdk.management
  * @library /test/lib
- * @run junit/othervm/native -Xint --enable-native-access=ALL-UNNAMED SynchronizedNative
+ * @run junit/othervm/native/timeout=300 -Xint --enable-native-access=ALL-UNNAMED SynchronizedNative
```

Comment:
https://github.com/eclipse-openj9/openj9/pull/21700#issuecomment-2824429516 notes that `-Xcomp` affects performance: why do we need to alter the timeout for other cases?

Reply:
[openjdk24_j9_sanity.openjdk_x86-64_windows](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk24_j9_sanity.openjdk_x86-64_windows/37/) was timed out at `java/lang/Thread/virtual/SynchronizedNative.java#Xint`, [openjdk24_j9_sanity.openjdk_aarch64_mac](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk24_j9_sanity.openjdk_aarch64_mac/45/) was `java/lang/Thread/virtual/SynchronizedNative.java#Xcomp-TieredStopAtLevel1`. Hence increased the four variations instead.

Reply:
Have we seen the normal variant time-out (with neither `-Xint` nor `-Xcomp`)? What's the default time limit?

Reply:
This test had segmentation error before, not much test coverage yet.
`timeout=300` is often used when extra time out is required. (There are quite a few of them added by OpenJDK)
I think the default timeout value is `120`, actual time limit is also affected by another time factor as per platforms.